includes/data-events/connectors/ga4/class-ga4.php

```diff
@@ -179,7 +180,7 @@ private static function extract_cid_from_cookies() {
 				list( $version, $domain_depth, $cid ) = $cookie_pieces;
 			}
 		} else {
-			$cid = 555;
+			throw new \Exception( 'Missing _ga Cookie' );
```

Comment:
We shouldn't `throw` here. This is not running on the handler execution. It's behind the dispatch's body filter, which is synchronous and will throw a fatal to the user.

Reply:
Thanks!